zenodo/modules/spam/views.py

```diff
@@ -144,8 +162,13 @@ def _expand_users_info(results, include_pending=False):
         ).filter(User.id.in_(results.keys()))
     )
 
+    normalized_emails_list = Counter([
+        normalize_emails(user.email)
+        for user in user_data
+    ])
+
     for user in user_data:
-        if user.safelist or (not user.active and not include_pending):
+        if (user.safelist or not user.active) and not include_pending:
```

Comment:
This slightly changes the filtering, but looking at it again I see that it might have been wrong all along. Let's discuss IRL on how it should be.